components/wpt-results.html

```diff
@@ -284,6 +287,20 @@
       _dateFormat(dateStr) {
         return String(new Date(dateStr)).match(/^\w+ (\w+ \w+ \w+)/)[1]
       }
+
+      _testFileHSL(result) {
```

Comment:
Optionally, you could also call this _testResultStyle since it returns a CSS style rule string